packages/app-security/src/components/NotAuthorizedError.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Link } from "@webiny/react-router";
+import { css } from "emotion";
+import styled from "@emotion/styled";
+import Helmet from "react-helmet";
+import authErrorImg from "../admin/assets/images/SecureRouteError.svg";
+import { Typography } from "@webiny/ui/Typography";
+
+const ContentWrapper = styled("div")({
+    display: "block",
+    paddingTop: "15%",
+    textAlign: "center",
+    margin: "auto"
+});
+
+const styles = {
+    authErrorImgStyle: css({
+        width: "192px"
```

Comment:
I would add some bottom spacing here. For example, `padding-bottom: "24px"`

Reply:
Okay I will add that